noir_stdlib/src/hash/poseidon/bn254.nr

```diff
@@ -0,0 +1,9 @@
+// Instantiations of Poseidon constants, permutations and sponge for prime field of the same order as BN254
+mod perm;
+mod consts;
+
```

Comment:
Could you add a function which checks the security of the instantiation?
check_security(rate, width, security (in bits) {
//I think there is a function in stdlib to get the size in bits of the field
//so you could constrain something like: (n>1 being the size in bits)
(n-1)/2*(width-rate) > security
}